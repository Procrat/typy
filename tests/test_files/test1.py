def f(x):
    return g(x)


def g(x):
    x + 1
    ~x
    return x


print(f(5) + 6)
f(5).non_existent_method()
