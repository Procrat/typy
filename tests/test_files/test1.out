Int has no attribute non_existent_method
